Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
```
make distclean                                make: stack: Command not found
/bin/sh: 1: [: -ge: unexpected operator
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
make: stack: Command not found
/bin/sh: 1: [: -ge: unexpected operator
```
As said: please make sure, that stack is there before using it blindly!

Also wrt. all other stack related changes make sure, that it can still be build via launchpad (unless the PPA is not needed anymore)! Not sure, but IIRC, downloading something during the build is not allowed. You could do so by setting up your own ppa and try to build a simple example package using stack. When you have worked out, how this needs to be done, it needs to be "merged" into hets build process ...